.github/workflows/team-labeler.yml

```diff
@@ -1,4 +1,7 @@
-on: pull_request
+on:
+  pull_request:
+    types: [opened]
+
```

Comment:
Instead of running on every PR-related action, just run once when the PR is opened.